MREUnityRuntime/MREUnityRuntimeLib/PluginInterfaces/Behaviors/IButtonBehavior.cs

```diff
@@ -9,19 +9,6 @@ namespace MixedRealityExtension.PluginInterfaces.Behaviors
 	/// </summary>
 	public interface IButtonBehavior : ITargetBehavior
```

Comment:
Why does this interface still exist, if it's empty?

Reply:
It is to enforce that there needs to be a class backing a button behavior supplied to us.  There will be things added to this in the future for when we want to pass parameters down to the behavior to configure and or patch it.

---
In reply to: [434211722](https://github.com/microsoft/mixed-reality-extension-unity/pull/160#discussion_r434211722) [](ancestors = 434211722)